engines/scumm/script_v2.cpp

```diff
@@ -1282,16 +1282,16 @@ void ScummEngine_v2::o2_putActorInRoom() {
 	// After first using the blue crystal, a cutscene of the two Caponians plays (script-96),
 	// locking object 344 (which prevents the cutscene playing again) and setting Var[245] to 0x18.
 	// script-5 uses this variable to set the Caponian costume
-	// On first apperance after using the blue crystal, the Caponians now will have the disguise on
+	// On first appearance after using the blue crystal, the Caponians now will have the disguise on
 	//
 	// If you visit the spacecraft and ring the doorbell, Var[245] will be set to 0x1C (Disguise off)
 	// Using the blue crystal again, will result in the Caponian appearing without his disguise
 	// as Var[245] is never set back to 0x18. This WORKAROUND fixes the problem by ensuring
-	// Var[245] is set to have the Disguise on in most situations
+	// Var[245] is set to have the disguise on in most situations
```

Comment:
not sure about this one, considering line 1287 I think "Disguise on" and "Disguise off" are the two possible settings of the variable, so the capitalization is probably deliberate?